### DISCUSSION TITLE
Improve DAE-11 optimization

### DISCUSSION THREAD

**Discussion by kjayawar** [peterdsharpe/AeroSandbox#111] [Show and tell]
Hi,

I've been playing with the 2D airfoil optimization [tutorial](https://github.com/peterdsharpe/AeroSandbox/blob/master/tutorial/06%20-%20Aerodynamics/02%20-%20AeroSandbox%202D%20Aerodynamics%20Tools/02%20-%20NeuralFoil%20Optimization.ipynb) and realized the difference in polar is could be due to Re number mismatch at higher speeds. (Using Type1 polar instead of Type2)

I just changed the line
`Re = 500e3`
into 
`Re = 500e3/CL_multipoint_targets**.5`

And the optimization resulted in a better polar as shown below. (As per my understanding)

<img width="374" alt="overlay" src="https://github.com/peterdsharpe/AeroSandbox/assets/77035452/3db70324-62e0-4bdd-9bc8-fc52fca4a4e0">
<img width="374" alt="ReCL_polar" src="https://github.com/peterdsharpe/AeroSandbox/assets/77035452/f2e16cc8-89b7-446e-aac2-d014f426470e">

Please incorporate this diff into the tutorial if you think it is useful

Thanks
Kanishka

**Comment by peterdsharpe** [comment]
Hey Kanishka,

Good catch! I think you're spot-on with this observation - a Type 2 polar captures the design intent better here.

A minor tweak is to normalize the calculation about the single-point-design $C_L$ of 1.25, so:

$\mathrm{Re}_c = 500\mathrm{k} \cdot \left(\frac{C_L}{1.25}\right)^{-0.5}$

This change has been implemented in this commit: https://github.com/peterdsharpe/AeroSandbox/commit/c9ccf6b650f9020a54c13e86768c1f4d56627071

Thanks! :)
